### PR TITLE
patch to allow non-number endings, i.e. RCx

### PR DESCRIPTION
**Pull request by LegacyKing**

### CODE REVIEW

**Comment by grimreaper** [comment]
@LegacyKing I modified this PR to also fix some syntax. That said, LGTM.

**Comment by LegacyKing** [comment]
@grimreaper the original code WORKED for the RC5 release. I don't understand the syntax, you'd have to ask @javydreamercsw as he supplied it. I'm just putting it here so we can include it.

I cannot merge it if you've altered the syntax and no time to test if it breaks the release.

**Comment by grimreaper** [comment]
I believe it worked for RC5. I'm worried it'll break once the version stops being "RC".
Perhaps @javydreamercsw could comment?

**Comment by javydreamercsw** [comment]
It'll basically work for anything after the number.

Basically it tokenizes with the period. So 6.00.00XX

Turns into 6 00 00XX. For the last piece it does a substring of the first 2 characters.

File: code/gradle/release.gradle [hunk 1]
```diff
@@ -215,7 +214,6 @@ task downloadJRE doLast {
                         fileTree(dir: "${projectDir}/jre/${os}/").include("*.${extension}").each { simLib ->
                             copy {
                                 println "Unzip $simLib.name"
-                                def name = simLib.name - '.$extension'
```
Comment (javydreamercsw) [inline code/gradle/release.gradle]: I thought this was used somewhere else. Maybe it was just an output no longer there.

This should be easy to test. Just delete the JRE folder and call this task.

File: code/gradle/release.gradle [hunk 2]
```diff
@@ -110,8 +109,8 @@ ext {
     }
 
 	List result = plainVerNum.replaceAll('-DEV', '').tokenize('.')
-	installerVerNum = result.get(0).toInteger() + "." + result.get(1).toInteger() + "." +  result.get(2).toInteger() + ".0"
-}
+	installerVerNum = result[0].toInteger() + "." + result[1].toInteger() + "." + result[2].take(2).toInteger() + ".0"
+	}
```
Comment (javydreamercsw) [inline code/gradle/release.gradle]: This should be still correct.

**Pull request merged**